packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -303,6 +303,7 @@ export default function PageTemplatesTemplateParts( { postType } ) {
 			_fields.push( {
 				header: __( 'Description' ),
 				id: 'description',
+				gridDisplayDirection: 'column',
```

Comment:
Can we wait a bit before merging this? I'm AFK today, but saw the approval and here we introduce an API in fields tied to a specific view. We need to think this better and it's time to find a better solution that also covers the `width, min-width` props we have, that are similar cases(they are tied to table view).

Reply:
For sure, I won't merge until you've reviewed.

Reply:
Thank you! I pushed an update to move this option to layout config.